src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -72,7 +72,12 @@
   static final Pattern COMPLETED_ORDER_PATTERN =
       Pattern.compile("completed *order", CASE_INSENSITIVE);
   static final Pattern PRODUCT_EVENT_NAME_PATTERN =
-      Pattern.compile("((viewed)|(added)|(removed)) *product *.*", CASE_INSENSITIVE);
+      Pattern.compile("(((viewed)|(added)|(removed)) *product *.*)|"
```

Comment:
It seems both `verb action` and `action verb` are supported - shouldn't it just be one?